tools/azure-devops/win/build.ps1

```diff
@@ -66,6 +66,7 @@ try {
                 -DUA_ENABLE_PUBSUB:BOOL=ON `
                 -DUA_ENABLE_PUBSUB_DELTAFRAMES:BOOL=ON `
                 -DUA_ENABLE_PUBSUB_INFORMATIONMODEL:BOOL=ON `
+                -DUA_LOGLEVEL:INT=0 `
```

Comment:
Can you explain the background of this change a little?

Reply:
This change will be deleted
I wanted to enable more traces, because I think that PubSub is sometimes not working
I can see that the publisher writes the value to the buffer, but it's not received (sometimes)